tools/fix_auth/cli.rb

```diff
@@ -24,6 +24,7 @@ def parse(args, env = {})
         opt :databaseyml, "Rewrite database.yml", :type => :boolean, :short => "y", :default => false
         opt :db,       "Upgrade database",  :type => :boolean, :short => 'x', :default => false
         opt :legacy_key, "Legacy Key",      :type => :string, :short => "K"
+        opt :resilient, "Run through all records, even with errors", :type => :boolean, :short => nil, :default => false
```

Comment:
Maybe something like pg_restore's exit on error, but default it to true so it won't blindly continue on

```
 -e
--exit-on-error
```

https://www.postgresql.org/docs/9.6/static/app-pgrestore.html